src/content/getting-started/designers/designers.md

```diff
@@ -24,7 +24,7 @@ title: Designers
 <anchor-links>
 <ul>
     <li><a href="#carbon-design-kit">Carbon Design Kit</a></li>
-    <li><a href="#how-to-get-it">How to get it</a></li>
+    <li><a href="#how-to-get-it">How to get the kit</a></li>
```

Comment:
This capitalization is correct -- keel it lowercase when just referring to the "design kit" or "kit"